README.md

```diff
@@ -1022,18 +995,18 @@ that you used.
 that you want to give the `.ppm` file.  The `.ppm` file is needed by
 `vc_layers_from_ppm`.
 
-So the command you want to type will look something like:
+So, the command you want to type will look something like:
 ```
 vc_render -v [your volpkg directory] --input-mesh [your .obj file] --output-ppm [the name of the ppm file you want to create]
 ```
-You might need to use `--volume` to specify your volume as well, if your volpkg has more than one.
+You might need to use `--volume` to specify your volume as well, if your `.volpkg` has more than one.
```

Comment:
Sure.

Reply:
Could you let me know what your choice is here, please?
If you want to change it back to a **noun**, you can also suggest to make the change elsewhere.
(depends on your interpretation of each case)

I would suggest using "`.volpkg` file", adding "file" to make it a noun. It should be more comprehensive.
I assume it is still a file (extension).

Reply:
I like your suggestion, which I will modify to "`.volpkg` directory", since Volume Cartographer (and khartes) somewhat confusingly put file extensions on directory names.  Or if you think "folder" is more understandable than "directory", that is fine with me too.

Reply:
Directory is fine.